.changeset/polite-books-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+'@ag.ds-next/react': patch
+---
+
+date-picker: Fix DatePicker scrolling to the top of the page on first open.
```

Comment:
Should we add the other things that were fixed?

Reply:
Yep, sounds good.

- [ ] Add the list of fixes to the release notes